library/CoreBundle/Resources/config/orm_target_entities.yml

```diff
@@ -221,6 +221,8 @@ doctrine:
         Ivoz\Cgr\Domain\Model\RatingPlan\RatingPlan
```

Comment:
Rememeber to add Cgr namespace into ivozprovider/library/CoreBundle/Resources/config/doctrine.yml
  ```
orm:
    mappings:
      Cgr: ...
```